scripts/globals/survival_guide_map.lua

```diff
@@ -0,0 +1,1181 @@
+        groupMask = 4194304,
+        groupIndex = 23,
+        posX = 82,
+        posY = 1,
+        posZ = -68,
+        posRot = 64
+    },
+    [82] = {
+        value = "Bastok Market \\[S\\]",
+        zoneId = 87,
+        group = 3,
+        groupMask = 8388608,
+        groupIndex = 24,
+        posX = -246,
+        posY = 0,
+        posZ = 94,
+        posRot = 64
+    },
+    [89] = {
+        value = "Windurst Waters \\[S\\]",
+        zoneId = 94,
+        group = 3,
+        groupMask = 16777216,
+        groupIndex = 25,
+        posX = -57,
+        posY = -5,
+        posZ = 215,
+        posRot = 64
+    },
+    [76] = {
+        value = "Caedarva Mire",
+        zoneId = 79,
+        group = 3,
+        groupMask = 33554432,
+        groupIndex = 26,
+        posX = -658,
+        posY = -13,
+        posZ = 342,
+        posRot = 64
+    },
+    [75] = {
+        value = "Arrapago Reef",
+        zoneId = 54,
+        group = 3,
+        groupMask = 67108864,
+        groupIndex = 27,
+        posX = 473,
+        posY = -16,
+        posZ = -16,
+        posRot = 64
+    },
+    [73] = {
+        value = "Halvung",
+        zoneId = 62,
+        group = 3,
+        groupMask = 134217728,
+        groupIndex = 28,
+        posX = 501,
+        posY = 0,
+        posZ = 241,
+        posRot = 64
+    },
+    [45] = {
+        value = "Behemoth\'s Dominion",
+        zoneId = 127,
+        group = 3,
+        groupMask = 268435456,
+        groupIndex = 29,
+        posX = 338,
+        posY = 26,
+        posZ = -77,
+        posRot = 64
+    },
+    [49] = {
+        value = "Dragon\'s Aery",
+        zoneId = 154,
+        group = 3,
+        groupMask = 536870912,
+        groupIndex = 30,
+        posX = -60,
+        posY = -1,
+        posZ = -30,
+        posRot = 64
+    },
+    [54] = {
+        value = "Valley of Sorrows",
+        zoneId = 128,
+        group = 3,
+        groupMask = 1073741824,
+        groupIndex = 31,
+        posX = -163,
+        posY = -8,
+        posZ = 21,
+        posRot = 64
+    },
+    [63] = {
+        value = "Ifrit\'s Cauldron",
+        zoneId = 205,
+        group = 3,
+        groupMask = 2147483648,
+        groupIndex = 32,
+        posX = 87,
+        posY = 0,
+        posZ = -297,
+        posRot = 64
+    },
+    [21] = {
+        value = "Zeruhn Mines",
+        zoneId = 172,
+        group = 4,
+        groupMask = 1,
+        groupIndex = 1,
+        posX = -11,
+        posY = 0,
+        posZ = 5,
+        posRot = 64
+    },
+    [97] = {
+        value = "Eastern Adoulin",
+        zoneId = 257,
+        group = 4,
+        groupMask = 2,
+        groupIndex = 2,
+        posX = -51,
+        posY = 0,
+        posZ = -120,
+        posRot = 64
+    }
+}
+
+zoneIdToMenuID = {
+    [231] = {menuId = 0},
+    [234] = {menuId = 1},
+    [240] = {menuId = 2},
+    [243] = {menuId = 3},
+    [26] = {menuId = 4},
+    [50] = {menuId = 5},
+    [100] = {menuId = 6},
```

Comment:
(And maybe some spacing to line everything up. :wink:

Reply:
This is how VS code lined everything up which is based on this projects editor config.

Reply:
re-spacing this would be optional for you. editorconfig mostly gets this right, but it'll never be perfection (a lot is dependent on an editors interpretation of editirconfig) :smiley: